plugins/aspects/README.md

```diff
@@ -0,0 +1,22 @@
+# Aspects Plugins
```

Comment:
Our hope here was that this plugin wouldn't need to reside here, but rather would be able to live in one of the Aspects repos (platform-plugin-aspects or tutor-contrib-aspects). Is that possible to do?

Reply:
I agree. I wasn't sure where to put this as well, I was hoping @farhaanbukhsh would give me some ideas during our internal review :)

On the tech side, as long as can install it as an NPM package any place should be good. The main side-effect I can think of would be to publish yet another NPM package.

Reply:
In a discussion yesterday, @bmtcril  pointed out the segregation in responsibilities between two repos. Where anything related to LMS/CMS belongs to the platform-plugin-aspect while tutor-contrib-aspects will have all the configurations.

@tecoholic any thoughts?

Reply:
@farhaanbukhsh The separation of concerns sounds sensible to me.

@bmtcril Practically what would be appropriate in the case? Personally, I think publishing an NPM package like `@openedx/frontend-plugin-aspects` could be complementary to `platform-plugin-aspects`. It might mean it's own repo .Or maybe it shares a directory in "platform-plugin-aspects" repo? I am not clear on the logistics of this. Kindly advise. I will move the plugin code accordingly.

Reply:
@tecoholic @farhaanbukhsh I think what we've come up with on the Axim side is that this should probably live in its own repo (`frontend-plugin-aspects`) until we have a better way of combining front-end, back-end, and tutor plugins in one place. If that's good for you all I can create the new repo for you in the openedx org and make you the maintainers of it through the support portion of this contract. 

That way we don't have to deal with moving it later and setting up all of the automation, and I believe we can just grant you the permissions as the creators of the plugin (instead of going through the CC process).

Reply:
@bmtcril Great. Can you kindly create the repo and provide access to @farhaanbukhsh. As he is a CC, it should be fairly straightforward than my account without CC rights. He would be the primary reviewer for these changes from OpenCraft's side anyway.

Reply:
@bmtcril  gentle ping on creating the repo  😇

Reply:
I'm working on it now, it's going to start as a clone of frontend-template-application, but I know that won't be a perfect fit so feel free to adjust as necessary.